icepyx/core/query.py

```diff
@@ -688,17 +709,16 @@ def order_vars(self):
                 # DevGoal: check for active session here
                 if hasattr(self, "_cust_options"):
                     self._order_vars = Variables(
-                        self._source,
-                        auth = self.auth,
                         product=self.product,
+                        version = self._version,
+                        auth = self.auth,
                         avail=self._cust_options["variables"],
```

Comment:
```suggestion
                        avail=self._cust_options["variables"],
                        auth = self.auth,
```
Do you know if there's a "best practice" around having the vars in the order they are in the function's init (even if they're kwargs)? Or is that just my OCD? ; )